### PR TITLE
qa/tests: added ragweed coverage to stress-split* upgrade suites

### PR DESCRIPTION
**Pull request by yuriw**
Fixes: http://tracker.ceph.com/issues/40452
Fices: http://tracker.ceph.com/issues/40467
Signed-off-by: Yuri Weinstein <yweinste@redhat.com>

- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

### CODE REVIEW

**Comment by cbodley** [comment]
@yuriw in the existing parallel upgrade suites, it runs the `rgw_ragweed_prepare.yaml` part before the upgrade, and the `rgw_ragweed_check.yaml` one after. the stuff under `7-final-workload` happens after the upgrade, right?

**Comment by yuriw** [comment]
@cbodley I think I am messing "before upgrade" piece, add to '4-workload' as "before upgrade" step and to  '7-workload' as "after upgrade" step

File: qa/suites/upgrade/mimic-x/stress-split/4-workload/rgw_ragweed_prepare.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+meta:
+- desc: |
+   rgw ragweed `prepare` before upgrade
+workload:
```
[the remark this answers is not in the document]
Comment (yuriw) [reply qa/suites/upgrade/mimic-x/stress-split/4-workload/rgw_ragweed_prepare.yaml]: done

File: qa/suites/upgrade/mimic-x/stress-split/7-final-workload/rgw_ragweed_prepare.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+meta:
+- desc: |
+   rgw ragweed `check` after upgrade
+workload:
+  full_sequential:
+  - sequential:
```
[the remark this answers is not in the document]
Comment (yuriw) [reply qa/suites/upgrade/mimic-x/stress-split/7-final-workload/rgw_ragweed_prepare.yaml]: see edits, will this work @cbodley ?

[the remark this answers is not in the document]
Comment (cbodley) [reply qa/suites/upgrade/mimic-x/stress-split/7-final-workload/rgw_ragweed_prepare.yaml]: yeah, adding `ragweed check` stuff to rgw-swift.yaml makes sense. maybe just rename it to rgw.yaml?

**Comment by yuriw** [comment]
http://pulpito.front.sepia.ceph.com:80/yuriw-2019-06-19_20:22:24-upgrade:mimic-x:stress-split-nautilus-distro-basic-smithi/

**Comment by yuriw** [comment]
http://pulpito.ceph.com/yuriw-2019-06-20_16:30:44-upgrade:mimic-x:stress-split-nautilus-distro-basic-smithi/

**Comment by cbodley** [comment]
two successful jobs in http://pulpito.ceph.com/yuriw-2019-06-20_19:58:33-upgrade:mimic-x:stress-split-nautilus-distro-basic-smithi/

i checked the logs and verified that both the prepare and check stages were successful, and the radosgw log showed a prepare workload on `ceph version 13.2.6-4-g9537302` and a swift and ragweed check workload on `ceph version 14.2.1-551-gc9923ec`

:+1: :rocket: :tada:

**Pull request merged**